tfjs-backend-webgpu/src/shader_preprocessor.ts

```diff
@@ -203,6 +205,18 @@ const SAMPLING_SNIPPETS = `
     return int(dot(coords, ivec4(
       shape.y * shape.z * shape.w, shape.z * shape.w, shape.w, 1.)));
   }
+
+  int getGlobalIndex() {
+    if (dispatchSize.y == 1 && dispatchSize.z == 1)
+    {
```

Comment:
for style consistency
```suggestion
    if (dispatchSize.y == 1 && dispatchSize.z == 1) {
```